doc/guile-libnotify.texi

```diff
@@ -116,7 +117,128 @@ If you wish to install you can @command{make install}, too. It may require
 sudo.
 
 @node How to use
-@chapter How to use
+@chapter How to use Guile-Libnotify
+
+Guile-Libnotify provides two main ways to send desktop notifications: a high-level interface using the @code{notify-send} function and a low-level interface using the libnotify API. This chapter explains both approaches.
+
+@menu
+* Using notify-send::           High-level interface for sending notifications
```

Comment:
I think there are repeated menus, I would siimplify this instead.
The resulting info file is kind of broken because of this.